lib/Cargo.toml

```diff
@@ -40,8 +40,15 @@ whoami = "1.5"
 tar = "0.4.42"
 flate2 = "1.0.34"
 file-owner = "0.1.2"
-gix = { version = "0.68.0", features = ["blocking-http-transport-reqwest-rust-tls", "blocking-network-client"] }
+gix = { version = "0.68.0", features = [
+    "blocking-http-transport-reqwest-rust-tls",
```

Comment:
> Pulled the branch to run it. With just some initial running, I came across this being spit out in the comtrya output.

Just saw this, without testing; it's likely because of the use of blocking calls, I didn't test file downloads or git clones, but here and reqwest both use the blocking protocol and that won't work with async. Probably just changing these will fix it, I'll take a look and correct when I get a chance.

Reply:
gix works with async_std not regular async